Source/RMG/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ set(RMG_SOURCES
     UserInterface/Dialog/SettingsDialog.qrc
     UserInterface/Dialog/RomInfoDialog.cpp
     UserInterface/Dialog/RomInfoDialog.ui
+    UserInterface/Dialog/AboutDialog.ui
+    UserInterface/Dialog/AboutDialog.cpp
```

Comment:
Please swap these around like with `RomInfoDialog`, the `cpp` file first and `ui` file second.